ci/do_ci.sh

```diff
@@ -110,6 +110,8 @@ else
   TEST_TARGETS="${COVERAGE_TEST_TARGETS} @com_googlesource_quiche//:ci_tests"
 fi
 
+BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --local_cpu_resources=${NUM_CPUS}"
```

Comment:
Just put those options as BAZEL_BUILD_EXTRA_OPTIONS or in `user.bazelrc`. We're trying to minimize options we modify in do_ci.sh / build_setup.sh . NUM_CPUS and local_cpu_resources are both defaulted to what CPU cores are visible so this does nothing.

Reply:
Ok. I got it. I will close this PR. :)